specification/storage/data-plane/Microsoft.FileStorage/preview/2020-02-10/file.json

```diff
@@ -5285,6 +5285,32 @@
                 "name": "Range"
             }
         },
+        "ClearRange": {
+            "type": "object",
+            "required": [
+                "Start",
+                "End"
+            ],
+            "properties": {
+                "Start": {
+                    "type": "integer",
+                    "format": "int64",
+                    "xml": {
+                    "name": "Start"
+                }
+            },
+            "End": {
+                "type": "integer",
+                "format": "int64",
+                "xml": {
+                    "name": "End"
+                }
+            }
```

Comment:
is the indentation correct? can you double check

Reply:
I think I fixed it.